docs/_documentations/mdt-eclipse-firstproject.md

```diff
@@ -1,46 +1,48 @@
 ---
 layout: docs
-title: "Codewind for Eclipse"
-description: "Codewind for Eclipse"
+title: "Creating your first Eclipse Codewind project"
+description: "Creating your first Eclipse Codewind project"
 keywords: introducing, introduction, overview, what is, tools, vscode, visual, studio, code, java, microprofile, spring, node, nodejs, node.js, javascript, Codewind for VS Code, tools, view, debug, integrate, open a shell session, toggle auto build, manually build, scope VS Code workspace, disable, enable, delete
 duration: 1 minute
 permalink: mdt-eclipse-firstproject
 type: document
 order: 2
 parent: mdt-eclipse-overview
 ---
-# Creating your first project
+# Creating your first Eclipse Codewind project
 <br/>
-To create your first project, right-click the **Local [Running]** item and select **Create New Project...**
+To create your first project:
+
+1\. Right-click the **Local [Running]** item >  select **Create New Project...**.
 
 ![](dist/images/eclipsecreateproject1.png){:width="800px"}
 
-This will display the project creation screen
+2\. Creating a new project displays the project creation screen.
 
 ![](dist/images/eclipsecreateproject2.png){:width="800px"}
 
-Enter a name for the project, change the location if you wish and then scroll the list of templates until you see **Node.js Express**
+3\. Enter a name for the project > change the location if you want > scroll through the list of templates until you see **Node.js Express**. 
```

Comment:
Change to: `Enter a name for the project and change the location if you want. Scroll through the list of templates until you see **Node.js Express**.`